### DISCUSSION TITLE
Toggle broken on cmd/clink (again)

### DISCUSSION THREAD

**Discussion by mjcarman** [JanDeDobbeleer/oh-my-posh#5642] [Troubleshoot] [answered]
I reported `omp toggle` not working in cmd in #4959. It was fixed under #4965 but is broken again, presumably by commit [68a1e89](https://github.com/JanDeDobbeleer/oh-my-posh/commit/68a1e89f291b9bdf5eeaeea658d0102e51562f08) in [v23.13.2](https://github.com/JanDeDobbeleer/oh-my-posh/releases/tag/v23.13.2). It worked in v23.13.1 but does not work in v23.13.3. (I have screenshots, but they're failing to upload at the moment.)

**Comment by JanDeDobbeleer** [comment, marked as answer]
This is indeed a bug. Will fix.

**Reply by JanDeDobbeleer** [reply]
See #5643

**Reply by mjcarman** [reply]
Success! Confirmed working again in v23.13.4
